docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -74,82 +64,260 @@ brief interruptions.
-must have a rolling average Download speed of at least 100Mbps **AND** a rolling average Upload
-speed of at least 10Mbps **AND** a rolling average Latency of 50ms or less.
+- Speed Test Results are put into Tiers based on the minimum value of each Download, Upload and
+  Latency (logical `AND`).
+- Speed Test results that do not meet the minimum requirements for any Download, Upload, or Latency
+  are considered to have `Failed` and are not eligible for Rewards until the Speed Test Average is
+  improved (logical `OR`).
 
-A 5G Hotspot will automatically perform a Speed Test approximately every 12 hours with the results
-being averaged by the MOBILE PoC Oracle over the previous `6` tests, depending on the availability
-of the data. This moving average is reset following a speed test gap of `>12 hours` and a minimum of
-`2` Speed Test results are required to be eligible for Genesis Rewards.
+When combined with existing
+[Small Cell Type Reward Scale Mulitpliers](./mining-mobile#small-cell-radio-reward-scaling) the
+Rewards Scaling Table is as follows:
 
-:::note Moving Average
+| Speed Test         | Acceptable | Degraded |  Poor   |  Fail   |
+| ------------------ | :--------: | :------: | :-----: | :-----: |
+| High Power Outdoor |  `4.000`   | `2.000`  | `1.000` | `0.000` |
+| Outdoor            |  `2.500`   | `1.250`  | `0.625` | `0.000` |
+| Indoor             |  `1.000`   | `0.500`  | `0.250` | `0.000` |
 
-The "Moving Average", also known as a "Rolling Average", is the average of at least 2 but no more
-than 6 most recent speed test results for each download, upload, and latency.
+**Example 1: All Acceptable**
 
-This means that when more than 6 Speed Test results are available, results that came before 6th most
-recent test won't be considered in the calculation.
+```bash
+Download    200 ->  `Acceptable`
+Upload       40 ->  `Acceptable`
+Latency      15 ->  `Acceptable`
+---------------------------------
+Speed Test Tier ->  `Acceptable`  -> 1.0x Multiplier
+```
 
-:::
+**Example 2: Degraded Upload**
+
+```bash
+Download    200 ->  `Acceptable`
+Upload        6 ->  `Degraded`
+Latency      15 ->  `Acceptable`
+---------------------------------
+Speed Test Tier ->  `Degraded`    -> 0.5x Multiplier
+```
+
+**Example 3: Poor Download, Degraded Upload**
+
+```bash
+Download     40 ->  `Poor`
+Upload        6 ->  `Degraded`
+Latency      15 ->  `Acceptable`
+---------------------------------
+Speed Test Tier ->  `Poor`        -> 0.25x Multiplier
+```
+
+**Example 4: Failed Latency**
+
+```bash
+Download    200 ->  `Acceptable`
+Upload       40 ->  `Acceptable`
+Latency     150 ->  `Fail`
+---------------------------------
+Speed Test Tier ->  `Fail`        -> 0.0x Multiplier
+```
+
+### Statistics on Explorer
+
+Speed Test metrics can be viewed on the 5G Hotspot's page on Explorer under the `5G Statistics` tab.
+
+Here are some examples of each status and a brief description of each.
```

Comment:
Also below screens are for derived data. First iteration is to update the Explorer to displat info only screens, you can see them here https://github.com/helium/explorer/issues/1059 and in figma.